src/NuGet.Core/NuGet.PackageManagement/NuGet.PackageManagement.csproj

```diff
@@ -17,16 +17,16 @@
   
   <ItemGroup>
     <PackageReference Include="Microsoft.Web.Xdt" Version="2.1.1" />
+    <PackageReference Include="System.IO.Compression" Version="4.1.0" />
```

Comment:
As you've added this in `NuGet.Common` it should be available in all projects due to transitive nature of package dependencies.